template.yaml

```diff
@@ -27,18 +27,18 @@ Parameters:
   secretName:
     Type: String
     Default: "nhs-hans-staging"
+  vpcId:
+    Type: String
+  vpceId:

```

Comment:
Internal and External VPCs? I'd give a more verbose var name tbh. Single character difference with no description feels easy to miss.

Reply:
Just seen in the context of the other code that this is referring to the vpc endpoint, but it confusion around the var name is demonstrated I think...

Reply:
Yep - made more verbose!